apps/server/src/modules/migration/controller/dto/page-type.query.param.ts

```diff
@@ -0,0 +1,17 @@
+import { IsMongoId } from 'class-validator';
+
+export enum PageTypes {
+	START_FROM_NEW_SYSTEM = 'start_from_new_system',
+	START_FROM_OLD_SYSTEM = 'start_from_old_system',
+	START_FROM_OLD_SYSTEM_MANDATORY = 'start_from_old_system_mandatory',
+}
+
+export class PageContentQueryParams {
+	pageType!: PageTypes;
+
+	@IsMongoId()
+	sourceSystem!: string;
```

Comment:
I don't think it is a good idea to let the user decide, what source and what target system he wants to have. especially not on a public endpoint.

Let's talk